spec/requests/api/v1/moves_controller_show_spec.rb

```diff
@@ -1,62 +1,86 @@
 # frozen_string_literal: true
 
-require 'rails_helper'
+require 'swagger_helper'
 
 RSpec.describe Api::V1::MovesController, with_client_authentication: true do
-  let(:headers) { { 'CONTENT_TYPE': content_type }.merge(auth_headers) }
   let(:content_type) { ApiController::CONTENT_TYPE }
-  let(:response_json) { JSON.parse(response.body) }
-
-  let(:resource_to_json) do
-    JSON.parse(ActionController::Base.render(json: move, include: MoveSerializer::INCLUDED_ATTRIBUTES))
-  end
-
-  let(:detail_404) { "Couldn't find Move with 'id'=UUID-not-found" }
-
-  describe 'GET /moves/{moveId}' do
-    let(:schema) { load_json_schema('get_move_responses.json') }
-
-    let!(:move) { create :move }
-    let(:move_id) { move.id }
-
-    before do
-      allow(Moves::NomisSynchroniser).to receive(:new)
-      get "/api/v1/moves/#{move_id}", headers: headers
-    end
-
-    context 'when successful' do
-      it_behaves_like 'an endpoint that responds with success 200'
-
-      it 'returns the correct data' do
-        expect(response_json).to eq resource_to_json
+  let(:move) { create :move }
+  let(:moveId) { move.id }
```

Comment:
Why the capitalization change? In Ruby symbols, methods and variables are always snake_case - https://github.com/rubocop-hq/ruby-style-guide#snake-case-symbols-methods-vars

Reply:
This isn't a capitalisation change – it's porting the existing swagger document and that uses camel case throughout.

That said, I'm happy to change this one, and we can keep consistency for all new tests. At some point we'll need to back-fill the tests for the existing endpoints so we can fix it up then.